src/moveit_commander/planning_scene_interface.py

```diff
@@ -105,6 +105,14 @@ def __make_mesh(self, name, pose, filename, scale = (1, 1, 1)):
         pyassimp.release(scene)
         return co
     
+    def __make_existing(self, name):
+        """
+        Create an empty Collision Object, used when the object already exists 
+        """
+        co = CollisionObject()
+        co.id = name
```

Comment:
I'm not very familiar with the python interfaces but I looked around and that seemed right